docs/src/app/components/pages/components/auto-complete.jsx

```diff
@@ -0,0 +1,80 @@
+const React = require('react');
+
+const { AutoComplete } = require('material-ui');
+const ComponentDoc = require('../../component-doc');
+const CodeExample = require('../../code-example/code-example');
+
+const Code = require('auto-complete-code');
+
+class AutoCompletePage extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+
+    return (
+      <ComponentDoc
+        name="Auto Complete"
+        componentInfo={[{
```

Comment:
Could you add more information as for instance here https://github.com/callemall/material-ui/blob/master/docs/src/app/components/pages/components/dialog.jsx#L31